mqtt/mqttd/src/main.rs

```diff
@@ -7,13 +7,27 @@ use tracing_subscriber::{fmt, EnvFilter};
 
 use mqttd::broker;
 
+const BROKER_LOG_LEVEL_ENV: &str = "BROKER_LOG";
+const HUB_LOG_LEVEL_ENV: &str = "RuntimeLogLevel";
+
 #[tokio::main]
 async fn main() -> Result<()> {
+    let mut log_level = EnvFilter::try_from_env(BROKER_LOG_LEVEL_ENV);
```

Comment:
```suggestion
    let log_level = EnvFilter::try_from_env(BROKER_LOG_LEVEL_ENV)
        .or_else(|_| EnvFilter::try_from_env(EDGE_HUB_LOG_LEVEL_ENV))
        .unwrap_or_default();
```
it will try BROKER_LOG -> RuntimeLogLevel -> RUST_LOG